app/functional_tests_fixtures/__init__.py

```diff
@@ -0,0 +1,460 @@
+import os
+from uuid import uuid4
+
+import boto3
+from flask import Blueprint, current_app
+from sqlalchemy.exc import NoResultFound
+
+from app.constants import (
+    EDIT_FOLDER_PERMISSIONS,
+    EMAIL_AUTH,
+    EXTRA_LETTER_FORMATTING,
+    INBOUND_SMS_TYPE,
+    LETTER_TYPE,
+    SECOND_CLASS,
+    SEND_EMAILS,
+    SEND_LETTERS,
+    SEND_TEXTS,
+    VIEW_ACTIVITY,
+)
+from app.dao.annual_billing_dao import (
+    set_default_free_allowance_for_service,
+)
+from app.dao.api_key_dao import (
+    get_model_api_keys,
+    save_model_api_key,
+)
+from app.dao.inbound_numbers_dao import (
+    dao_get_inbound_number_for_service,
+    dao_set_inbound_number_to_service,
+)
+from app.dao.organisation_dao import (
+    dao_create_organisation,
+    dao_get_organisation_by_id,
+    dao_get_organisations_by_partial_name,
+    dao_update_organisation,
+)
+from app.dao.permissions_dao import permission_dao
+from app.dao.service_email_reply_to_dao import (
+    add_reply_to_email_address_for_service,
+    dao_get_reply_to_by_service_id,
+)
+from app.dao.service_inbound_api_dao import get_service_inbound_api_for_service, save_service_inbound_api
+from app.dao.service_permissions_dao import (
+    dao_add_service_permission,
+    dao_fetch_service_permissions,
+)
+from app.dao.service_sms_sender_dao import (
+    dao_add_sms_sender_for_service,
+    dao_get_sms_senders_by_service_id,
+)
+from app.dao.services_dao import (
+    dao_add_user_to_service,
+    dao_create_service,
+    dao_fetch_service_by_id,
+    get_services_by_partial_name,
+)
+from app.dao.templates_dao import dao_create_template, dao_get_all_templates_for_service
+from app.dao.users_dao import get_user_by_email, save_model_user
+from app.models import (
+    InboundNumber,
+    Organisation,
+    Permission,
+    Service,
+    ServiceEmailReplyTo,
+    ServiceInboundApi,
+    User,
+)
+from app.schemas import api_key_schema, template_schema
+
+functional_tests_fixtures_blueprint = Blueprint("functional_test_fixtures", __name__)
+
+
+@functional_tests_fixtures_blueprint.cli.command("apply")
+def apply():
```

Comment:
i think we could do with a docstring here with details on:

* how this function relates to the functional tests
* when this function gets invoked, and how it's used
* how and when we need to keep this up to date, with regards to all of:
  * functional test changes
  * env var changes
  * db schema changes

and anyhting else worth mentioning

Reply:
I've added a docstring.